src/apps/api/urls.py

```diff
@@ -6,10 +6,12 @@
 from rest_framework.permissions import AllowAny
 
 from api.views.competitions import front_page_competitions
-from .views import analytics, competitions, datasets, profiles, leaderboards, submissions, tasks, queues
+from .views import analytics, competitions, datasets, profiles, leaderboards, submissions, tasks, queues, benchmarks
 
 
 router = SimpleRouter()
+router.register('benchmarks', benchmarks.BenchmarkViewSet)
+router.register('benchmarks_status', benchmarks.BenchmarkCreationTaskStatusViewSet)
```

Comment:
why don't we keep existing viewsets and pass a flag/argument to filter by `competition_type`?

Reply:
wow, that will be great, need to pass a flag/argument from front end side

Reply:
:)